base/stats.go

```diff
@@ -1,13 +1,49 @@
 package base
 
-import "expvar"
+import (
+	"expvar"
+	"time"
+)
 
 var StatsExpvars *expvar.Map = expvar.NewMap("syncGateway_stats")
 var TimingExpvars SequenceTimingExpvar
+var perDbStatNames = []string{
+	"changesFeeds_active",
+	"changesFeeds_total",
+	"requests_active",
+	"requests_total",
+	"revisionCache_hits",
+	"revisionCache_misses",
+	"simpleChanges_active",
+	"simpleChanges_total",
+	"subChanges_active",
+	"subChanges_total",
+	"vectorChanges_active",
+	"vectorChanges_total"}
+var globalStatNames = append(perDbStatNames, []string{
+	"bulkApi.BulkDocsPerDocRollingMean",
+	"bulkApi.BulkDocsRollingMean",
+	"bulkApi.BulkGetPerDocRollingMean",
+	"bulkApi.BulkGetRollingMean",
+	"goroutines_highWaterMark",
+	"handler.CheckAuthRollingMean",
+	"indexReader.getChanges.Count",
+	"indexReader.getChanges.Time",
+	"indexReader.getChanges.UseCached",
+	"indexReader.getChanges.UseIndexed",
+	"indexReader.numReaders.OneShot",
+	"indexReader.numReaders.Persistent",
+	"indexReader.pollPrincipals.Count",
+	"indexReader.pollPrincipals.Time",
+	"indexReader.pollReaders.Count",
+	"indexReader.pollReaders.Time",
+	"indexReader.seqHasher.GetClockTime",
+	"indexReader.seqHasher.GetHash"}...)
 
 const (
 	KTimingExpvarVbNo      = 0
 	KTimingExpvarFrequency = 200
+	statLogInterval        = time.Minute * 1
```

Comment:
This feels too aggressive.  Can we set this to something like 5 minutes, and have a config option to tune it?

Reply:
Happy to add a config option (in fact, was already planning it ;)), but picked this as a nice round number that's a similar resolution to what Server logs